addons/survey/views/survey_templates_statistics.xml

```diff
@@ -113,15 +113,14 @@
             </t>
```

Comment:
One useless `/` in your PR link in your commit message
> PR: //pull/77620

Also for the fix, it's nice to add the commit responsible of the bug (git blame is your friend)

Here it's: fb7b30b2e1fe07e94e8ec689912fb3b07284d10e